CMakeLists.txt

```diff
@@ -332,6 +332,8 @@ protocolnew("staging/linux-drm-syncobj" "linux-drm-syncobj-v1" false)
 protocolnew("staging/xdg-dialog" "xdg-dialog-v1" false)
 protocolnew("staging/single-pixel-buffer" "single-pixel-buffer-v1" false)
 
+protocolnew("subprojects/hyprland-protocols/protocols" "hyprland-input-capture-v1"
```

Comment:
why is there an empty line above?

there should be an empty line only to separate wayland from w-ps

```cpp
protocolnew(...)
protocolnew(...)
protocolnew(...)

protocolwayland()
```